src/guiapi/include/display_math_helper.h

```diff
@@ -0,0 +1,94 @@
+//display_math_helper.h
+//basic mathematical operations for display
+#pragma once
+
+#include <inttypes.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif //__cplusplus
+
+__attribute__((used)) inline uint16_t swap_ui16(uint16_t val) {
```

Comment:
I don't like the non-standard attributes here, but the same code-magic was done in st77xxxxx before.